### PR TITLE
Ubi cuda

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: Dockerfile.redhat [hunk 1]
```diff
@@ -127,19 +130,74 @@ RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.n
...
+ARG ov_contrib_org=openvinotoolkit
 ARG ov_use_binary=1
 ARG DLDT_PACKAGE_URL
 ARG TEMP_DIR=/tmp/openvino_installer
+ARG CMAKE_BUILD_TYPE=Release
+
+# hadolint ignore=DL3003
+RUN if [[ "$sentencepiece" == "1" || "$NVIDIA" == "1" ]] ; then true ; else exit 0 ; fi ; git clone https://github.com/$ov_contrib_org/openvino_contrib.git /openvino_contrib && cd /openvino_contrib && git checkout $ov_contrib_branch && git submodule update --init --recursive
+
+################### BUILD OPENVINO FROM SOURCE - buildarg ov_use_binary=0  ############################
```
Comment (dkalinowski) [inline Dockerfile.redhat:184]: https://github.com/openvinotoolkit/model_server/blob/main/docs/build_from_source.md#ov_use_binary
We still document OV_USE_BINARY parameter with note: Warning: Maintained only for Ubuntu base OS.
Please remove this statement

File: Dockerfile.redhat [hunk 2]
```diff
@@ -127,19 +130,74 @@ RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.n
...
+    yum -y module disable python36 && \
+    yum install -y  \
+        libzstd-devel \
+        libcudnn8-8.6.0.163-1.cuda11.8 \
+        libcudnn8-devel-8.6.0.163-1.cuda11.8 \
+        libcutensor1-1.6.1.5-1 \
+        libcutensor-devel-1.6.1.5-1 \
+        cuda-cudart-devel-11-8 && \
+    # ignore errors on hosts with older nvidia drivers
+    yum install -y cuda-11-8 || true && \
```
Comment (jasiu86) [inline Dockerfile.redhat:153]: Is it intentional? 
`|| true`

**Review by jasiu86 — APPROVED** [review]

File: Dockerfile.redhat (hunk 2 above)
Comment (atobiszei) [reply Dockerfile.redhat:153]: @dtrawins

**Review by atobiszei — APPROVED** [review]

File: Dockerfile.redhat (hunk 2 above)
Comment (dtrawins) [reply Dockerfile.redhat:153]: Yes, that is intentional for the case with older driver installed on the host causing conflicts on some on the nested dependencies

**Pull request merged**